2016-05/may-26.md

```diff
@@ -0,0 +1,62 @@
+## May 26 ([discuss](https://github.com/reactjs/core-notes/pull/17))
+
+### Attendees
+
+* [Ben](https://twitter.com/soprano) (React)
+* [Christopher](https://twitter.com/vjeux) (React Native)
+* [Dan](https://twitter.com/dan_abramov) (React)
+* [Jim](http://github.com/jimfb) (React)
+* [Keyan](https://twitter.com/keyanzhang) (React, intern)
+* [Paul](https://twitter.com/zpao) (React)
+* [Sebastian](https://twitter.com/sebmarkbage) (React)
+* [Shayne](https://github.com/shayne) (React Native)
+* [Tom](https://twitter.com/tomocchino) (React)
+
+### ES2016 Classes
+
+#### Class Property Initializers ([proposal](https://github.com/jeffmo/es-class-fields-and-static-properties))
+
+* ES classes for React components don’t feel good without them.
+* We haven’t used them internally at Facebook enough to push externally.
+* For now, we’ll push for more internal adoption of ES classes.
+* We don’t want to hinder the proposal by pushing too hard for it now.
+* After the proposal is further along, we can update our internal code and start pushing externally.
+
+#### Documentation and Tutorial
+
+* We need to figure out how to make the docs reflect the fact that you can use either `createClass` or ES classes.
+* Proposal: the docs will continue to show `createClass` as the default, but will include a switcher to show examples using classes.
+* Eventually, we will flip the default.
+* In the tutorial we’ll start out introducing functional components (using function declarations).
+* [Dan](https://twitter.com/dan_abramov) and [Paul](https://twitter.com/zpao) will work on this.
```

Comment:
If you need any help migrating the examples, don't hesitate to contact me. I would be glad to help 👍